src/main/groovy/com/vherasymenko/avro/decoder/inbound/MessageListener.groovy

```diff
@@ -0,0 +1,24 @@
+package com.vherasymenko.avro.decoder.inbound
+
+import com.vherasymenko.avro.decoder.core.AvroDecoderPort
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.cloud.stream.annotation.EnableBinding
+import org.springframework.cloud.stream.annotation.StreamListener
+import org.springframework.cloud.stream.messaging.Sink
+import org.springframework.stereotype.Component
+
+/**
+ * The gateway to the messaging system.
+ */
+@Component
+@EnableBinding( Sink )
```

Comment:
It's possible to create your own type of binding, but I think there is no reason to do it here.
There three common type of bindings in Sping: `Sourse` for output channel, `Sink` for input one and `Processor` that extends them both.